packages/main/src/TabContainer.ts

```diff
@@ -469,25 +474,20 @@ class TabContainer extends UI5Element {
 	}
 
 	onEnterDOM() {
-		// eslint-disable-next-line
-		// TODO: handleResize is promise, it shouldn't be
-		ResizeHandler.register(this._getHeader(), this._handleResize); /* eslint-disable-line */
+		ResizeHandler.register(this._getHeader(), this._handleResizeBound);
 	}
 
 	onExitDOM() {
-		// eslint-disable-next-line
-		// TODO: handleResize is promise, it shouldn't be
-		ResizeHandler.deregister(this._getHeader(), this._handleResize); /* eslint-disable-line */
+		ResizeHandler.deregister(this._getHeader(), this._handleResizeBound);
 	}
 
-	async _handleResize() {
+	_handleResize() {
 		if (this.responsivePopover && this.responsivePopover.opened) {
 			this.responsivePopover.close();
 		}
-		this._updateMediaRange();
 
-		await renderFinished(); // await the tab container to have rendered its representation of tabs
-		this._setItemsForStrip();
+		this._updateMediaRange();
```

Comment:
You can add a parameter to this method and pass the offsetWidth. There is currently no difference between this.getDomRef().offsetWidth and this.offsetWidth (although they are two different elements - one from the light dom and the other from the shadow dom)

Reply:
Done